### ISSUE TITLE
[ENH]: revised tableau colors

### ISSUE THREAD

**Issue by nschloe** [matplotlib/matplotlib#21840]
### Problem

The default mpl color cycle (_Tableau10_) has been revised upstream in 2016, see [here](https://www.tableau.com/about/blog/2016/7/colors-upgrade-tableau-10-56782). This should probably be included in mpl.

![tableau10_palette](https://user-images.githubusercontent.com/181628/144405421-c8310c6f-d7b6-4e3d-8cfb-dc24f9084352.png)

**Comment by tacaswell** [comment]
I am 👎🏻 on this.  

First, we consider changing the color to be an API change (this is why we did mpl2.0).  Second we got permission to use the colors from Tableau so if we were to adopt the updated pallete under a new name ('muted-tab10', 'tab10-v2', 'tab10-16'?) we would want to make sure we got permission again.

As a side note, we don't ship `parula` because MATHWORKS considers it their IP, explicitly told us we could not ship it, and we have no interest in finding out if they would be willing to sue or if that IP claim would actually hold up in court.
